mycroft/filesystem/__init__.py

```diff
@@ -30,7 +31,13 @@ def __init__(self, path):
     def __init_path(path):
         if not isinstance(path, str) or len(path) == 0:
             raise ValueError("path must be initialized as a non empty string")
-        path = join(expanduser('~'), '.mycroft', path)
+
+        # Migrate from the old location if it still exists
+        old_path = join(expanduser('~'), '.mycroft', path)
+        if isdir(old_path):
+            os.rename(old_path, path)
```

Comment:
I changed this to `shutils.move`, but I think the real issue was that `path` was not pointing to the correct place (since it hasn't been concatenated with the XDG config dir). Not technically necessary, but shutils.move is more safe than os.rename since it works across mount points.